_posts/en/newsletters/2024-03-20-newsletter.md

```diff
@@ -0,0 +1,160 @@
+  it on signet/inquisition."
+
+  Russell O'Connor, developer of the [Simplicity][topic simplicity]
+  language that may also one day be considered as an alternative
+  consensus scripting language, [replied][oconnor lisp] with some
+  comparisons between Bitcoin's current Script language, Simplicity, and
+  Chia/BTC Lisp.  He concludes, "Simplicity and the [Chia Lisp Virtual
+  Machine] clvm are both low level languages that are meant to be easy
+  for machines to evaluate, which causes tradeoffs that make them hard
+  for humans to read. They are intended to be the compiled from some
+  different, human-readable, non-consensus-critical language.
+  Simplicity and the clvm are different ways of expressing the same old
+  things: fetching data from an environment, tupling up bits of data,
+  running conditional statements, and a whole bunch of primitive
+  operations of some sorts.  [...] Since we want this [split between
+  efficient low-level consensus language and high-level non-consensus
+  comprehensible language] regardless, the details of the low-level
+  language become somewhat less important. I.e., with some effort, your
+  high level BTC lisp language could probably be translated/complied to
+  Simplicity [...] Similarly, wherever the design of [Simplicity-based]
+  Simphony [high-level non-consensus language] ends up, it can probably
+  be translated/complied your low level BTC lisp language, with each
+  translator/compiler language pair offering different potential
+  complexity/optimization opportunities."
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.1rc1][] is a release candidate for a maintenance release
+  of the network's predominant full node implementation.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
+  version of the network's predominant full node implementation.
+
+<!-- FIXME:harding to update on Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until about six months after the release of the upcoming version 27.*
```

Comment:
Not sure I'm a fan of this note, or mentioning specifically Bitcoin Core and not the others. How about permanently amending the introductory paragraph like this?

"These changes are expected to be included in future releases of the respective projects."

(Although this is a bit inaccurate since BIPs, BOLTs and BINANAs don't have releases.)

Reply:
Previous discussion here: https://github.com/bitcoinops/bitcoinops.github.io/pull/395#discussion_r420143915

We haven't included this note for a while, but I saw some confusion on Twitter about it, so I decided to re-instate it.  I don't think the note generalizes, as only Bitcoin Core has this behavior; see this quote from the previous discussion:

> AFAICT, Bitcoin Core is the only project we monitor that branches from master prior to a major release. (Indeed, I'm unable to think of any other free software project I've followed that does that.)

I'm happy to discuss further, but I suggest we include the note this week as we've included it many times before.